src/component/marker/MarkerView.ts

```diff
@@ -88,6 +88,23 @@ abstract class MarkerView extends ComponentView {
         });
     }
 
+    leaveBlurSeries(seriesModelList: SeriesModel[]) {
```

Comment:
@pissang Can we merge `blurSeries` and `leaveBlurSeries`?

Reply:
I'm not sure how to do this. But I think these two may not be invoked in the same stage an it's better to keep them separated

Reply:
If merging means putting the shared logic in a utility method like `toggleSeriesBlur` and use it with a flag in `blurSeries` and `leaveBlurSeries`. It's a great idea and we should do this.